CHANGES.md

```diff
@@ -9401,7 +9377,7 @@ OpenSSL 0.9.x
 
    *Bodo Moeller; problem pointed out by Yutaka Oiwa (Research Center
    for Information Security, National Institute of Advanced Industrial
-   Science and Technology [AIST*, Japan)]
+   Science and Technology [AIST, Japan)]*
```

Comment:
Unbalanced parentheses; I would suggest changing

    Science and Technology [AIST, Japan)]*

to

    Science and Technology [AIST], Japan)*

Reply:
> Science and Technology [AIST], Japan)*
fixed.